models/omop/location.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
-    ROW_NUMBER() OVER () AS location_id
+    ROW_NUMBER() OVER (ORDER BY state_abbreviation, patient_city, patient_zip, p.patient_id) AS location_id
```

Comment:
i think we can remove patient ID since there's no patient ID column in the table?

in reviewing this, i realized this model has lots of problems.  rows in this table are supposed to be unique (https://ohdsi.github.io/CommonDataModel/cdm54.html#location). but this table has 1 row for every patient.

also, more importantly, this table isn't actually used for anything 😆 i'll file an issue to properly build this table and link it to PERSON and PROVIDER.

Reply:
yeah without an FK to person/provider it doesn't really make a difference.

i filed #82